spot_rl_experiments/spot_rl/utils/robot_subscriber.py

```diff
@@ -81,18 +86,36 @@ def __init__(self, spot=None, *args, **kwargs):
 
         rospy.loginfo(f"[{self.node_name}]: Robot subscribing has started.")
 
+    # Letting the user know about timeout to get images. This function lets the user know time taken to get the topics
+    def wait_for_images(self, subscriptions, timeout=30):
+        start_time = rospy.get_time()
+        while not all([self.msgs[s] is not None for s in subscriptions]):
+            if rospy.get_time() - start_time > timeout:
+                rospy.logwarn(f"[{self.node_name}]: Timeout waiting for image topics.")
+                break
```

Comment:
What happens after the timeout? don't we want to continue to wait for the image topics?

Reply:
this with a function that gets timeout, tells the user that there's a timeout and continues execution where it continues to subscribe to topics which are publishing.